sda/cmd/ingest/ingest.go

```diff
@@ -137,440 +149,445 @@ func main() {
+				if err := delivered.Reject(false); err != nil {
+					log.Errorf("failed to reject message for reason: (%s)", err.Error())
 				}
+			}
+		}
+	}()
 
-				file, err := inbox.NewFileReader(message.FilePath)
-				if err != nil { //nolint:nestif
-					log.Errorf("Failed to open file to ingest reason: (%s)", err.Error())
-					if strings.Contains(err.Error(), "no such file or directory") || strings.Contains(err.Error(), "NoSuchKey:") {
-						jsonMsg, _ := json.Marshal(map[string]string{"error": err.Error()})
-						if err := db.UpdateFileEventLog(fileID, "error", delivered.CorrelationId, "ingest", string(jsonMsg), string(delivered.Body)); err != nil {
-							log.Errorf("failed to set error status for file from message: %v, reason: %s", delivered.CorrelationId, err.Error())
-						}
-						// Send the message to an error queue so it can be analyzed.
-						fileError := broker.InfoError{
-							Error:           "Failed to open file to ingest",
-							Reason:          err.Error(),
-							OriginalMessage: message,
-						}
-						body, _ := json.Marshal(fileError)
-						if err := mq.SendMessage(delivered.CorrelationId, conf.Broker.Exchange, "error", body); err != nil {
-							log.Errorf("failed to publish message, reason: (%s)", err.Error())
-						}
-						if err = delivered.Ack(false); err != nil {
-							log.Errorf("Failed to Ack message, reason: (%s)", err.Error())
-						}
-
-						continue mainWorkLoop
-					}
-
-					if err = delivered.Nack(false, true); err != nil {
-						log.Errorf("Failed to Nack message, reason: (%s)", err.Error())
-					}
-
-					// Restart on new message
-					continue
-				}
+	<-forever
+}
 
-				fileSize, err := inbox.GetFileSize(message.FilePath, false)
-				if err != nil {
-					log.Errorf("Failed to get file size of file to ingest, reason: (%s)", err.Error())
-					// Nack message so the server gets notified that something is wrong and requeue the message.
-					// Since reading the file worked, this should eventually succeed so it is ok to requeue.
-					if err = delivered.Nack(false, true); err != nil {
-						log.Errorf("Failed to Nack message, reason: (%s)", err.Error())
-					}
-					// Send the message to an error queue so it can be analyzed.
-					fileError := broker.InfoError{
-						Error:           "Failed to get file size of file to ingest",
-						Reason:          err.Error(),
-						OriginalMessage: message,
-					}
-					body, _ := json.Marshal(fileError)
-					if err = mq.SendMessage(delivered.CorrelationId, conf.Broker.Exchange, "error", body); err != nil {
-						log.Errorf("failed to publish message, reason: (%s)", err.Error())
-					}
-
-					// Restart on new message
-					continue
-				}
+func (app *Ingest) cancelFile(correlationID string, message schema.IngestionTrigger) string {
+	fileUUID, err := app.DB.GetFileID(correlationID)
+	if err != nil {
+		log.Errorf("failed to get ID for file from message: %s, reason: %s", correlationID, err.Error())
```

Comment:
```suggestion
		log.Errorf("failed to get ID for file from message (correlationID: %s), reason: %v", correlationID, err)
```

Reply:
Otherwise, it is hard to know the value printed is correlationID without looking at the code.